ci-operator/step-registry/cucushift/e2e/cucushift-e2e-commands.sh

```diff
@@ -128,8 +128,8 @@ function filter_test_by_network() {
             networktag='@network-ovnkubernetes'
             ;;
         other)
-            networktag=''
-            ;;
+	    networktag='@other-cni'
```

Comment:
You will need to get the other PRs merged first, or else appending `and @other-cni` will cause no test case executed on `other` networks since there are no test case with that tag.